### PR TITLE
feat(groupchatsProgram): create a class to interact with the group chat program

### PR DESCRIPTION
**Pull request by nickjjang**
**What this PR does** 📖
Create a class to interact with the Group Chat program

**Which issue(s) this PR fixes** 🔨
AP-658

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: config.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ export const Config = {
     network: 'devnet',
     serverProgramId: 'FGdpP9RSN3ZE8d1PXxiBXS8ThCsXdi342KmDwqSQ3ZBz',
     friendsProgramId: 'BxX6o2HG5DWrJt2v8GMSWNG2V2NtxNbAUF3wdE5Ao5gS',
+    groupchatsPrograId: 'bJhvwTYCkQceANgeShZ4xaxUqEBPsV8e1NgRnLRymxs',
```
Comment (iltumio) [inline config.ts:48]: typo: groupchatsPrograId -> groupchatsProgra**m**Id

File: libraries/Solana/SolanaWallet.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { Keypair, PublicKey, Transaction } from '@solana/web3.js'
```
Comment (iltumio) [inline libraries/Solana/SolanaWallet.ts:1]: this file is no longer needed

**Review by iltumio — CHANGES_REQUESTED** [review]

**Comment by phillsatellite** [comment]
Tested: Seems the mockup for the groupchats in the sidebar is missing so user cant navigate into the group chat
Screen Shot 2022-02-18 at 8 00 56 PM (2)

File: libraries/Solana/GroupchatsProgram/GroupchatsProgram.ts [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+      },
+      signers: [payer],
+    })
+  }
+
+  /**
+   * @method invite
+   * Invite new user into Group
+   * @param groupId Group Id
+   * @param recipient: receipient address
```
Comment (stavares843) [inline libraries/Solana/GroupchatsProgram/GroupchatsProgram.ts:159]: typo - receipient

**Comment by WanderingHogan** [comment]
@nickjjang can you check why this PR breaks being able to see the hard coded group chats? It doesn't look like you altered anything in the ui at all.

**Review by phillsatellite — APPROVED** [review]

**Pull request merged**